setup.cfg

```diff
@@ -12,7 +11,9 @@ omit =
     testapp/migrations/*.py
 
 [flake8]
-exclude = .tox/*,build/*
+exclude = .tox/*,build/*,.eggs/*
+max-line-length = 88
+extend-ignore = E203, W503
 
 [wheel]
 universal = 1
```

Comment:
Universal is only needed for Python 2 + Python 3 compatibility, it can be removed for Python 3 only. 

See https://packaging.python.org/guides/distributing-packages-using-setuptools/#pure-python-wheels, https://github.com/PyCQA/pyflakes/pull/597#discussion_r536906408 and https://github.com/PyCQA/pyflakes/pull/597#discussion_r538432966
```suggestion
```

Reply:
Nice, you're right!